README.md

```diff
@@ -25,7 +28,7 @@ structures:
 * [Bifunctor](#bifunctor)
 * [Profunctor](#profunctor)
 
-<img src="figures/dependencies.png" width="863" height="347" />
+<img src="figures/dependencies.png" width="100%" />
```

Comment:
you can check rendered image [here](https://github.com/safareli/fantasy-land/tree/alternative)